examples/js/loaders/GLTFLoader.js

```diff
@@ -1408,20 +1589,18 @@ THREE.GLTFLoader = ( function () {
 			if ( node.jointName ) {
 
 				_node = new THREE.Bone();
+				_node.name = node.name !== undefined ? node.name : node.jointName;
 				_node.jointName = node.jointName;
 
 			} else {
 
 				_node = new THREE.Object3D();
+				if ( node.name !== undefined ) _node.name = node.name;
 
 			}
 
-			_node.name = node.name;
-
 			if ( node.extras ) _node.userData = node.extras;
 
-			_node.matrixAutoUpdate = false;
-
 			if ( node.matrix !== undefined ) {
```

Comment:
If `matrixAutoUpdate` is false,
translation/rotation/scale of this node won't be affected to world
when `scene.updateMatrixWorld(true)` is called.
So I think here should be `true(default)`.